### PR TITLE
UPDATE: Full-CRUD-Completed-For-Memebers_model

### PR DESCRIPTION
**Pull request by ValentinNM**
Hi
UPDATE: Full-CRUD-Completed-For-Memebers_model
awaiting confirmation for merging to **main**

### CODE REVIEW

File: src/resources/members/controller.js [hunk 1]
```diff
@@ -20,6 +20,102 @@ async function getAllMembers( req, res){
...
+
+    try { 
+
+        const newMember = await prisma.member.create({
+            data : { 
+                ...memeber
+            }
+        })
+
+        res.json({ theNewMemeber: newMember})
```
Comment (vherus) [inline src/resources/members/controller.js]: This means your frontend will be receiving a JSON object that looks like this:

```
{
    theNewMemeber: {
       // properties here
    }
}
```

That looks a little weird. I think you should just pass the new member object into json directly; `res.json(newMember)`

File: src/resources/members/controller.js [hunk 2]
```diff
@@ -20,6 +20,102 @@ async function getAllMembers( req, res){
...
+        const memeberNewDetails = await prisma.member.update({ 
+            where : {
+                id : parseInt(id)
+            }, 
+            data : { 
+                ...updatedData
+            }
+        })
+
+        res.json({ resp: memeberNewDetails})
```
Comment (vherus) [inline src/resources/members/controller.js]: Let's try to have some consistency with responses. This one will end up looking like this to the frontend:

```
{
    resp: {
        // properties here
    }
}
```

I'd just pass the `memeberNewDetails` directly again; `res.json(memeberNewDetails)`

**Review by vherus — CHANGES_REQUESTED** [review]
Looking good, but you've got some inconsistencies here with the way you're using `res.json`

Take a look at my comments and make those changes, then you're good to merge :)

File: src/resources/members/controller.js (hunk 1 above)
Comment (ValentinNM) [reply src/resources/members/controller.js]: oh, you're so right, did that on default mode based on reflex when checking `res` in the postman
I shall update accordingly

Comment (ValentinNM) [reply src/resources/members/controller.js]: done, this is soo cool 🤩 I can replay from VS code
once COVID showed up could not leave my house
now, once these features are here, I wont step outside my code editor 🥳

**Pull request merged**